Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "pulsar"
-version = "4.1.1"
-edition = "2018"
+version = "4.1.3"
+edition = "2021"
```

Comment:
these should be done in the release commit after merging.

Reply:
I agree

Reply:
I agree, but the version 4.1.3 is already published according to git tag. I will create another commit to do this change.

Reply:
I have remove the commit that do this change